### ISSUE TITLE
Default is no default

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#122]
**When creating a new document, its columns default is NOT the BrickStore default neither the User default:**

This is the new document, please see the “Cond.” column width:

![image](https://user-images.githubusercontent.com/32078594/105649486-c78b9200-5eb0-11eb-9796-33f1931f86a7.png)

This is the same document after I applied the “Default BrickStore” layout, compare the same column:

![image](https://user-images.githubusercontent.com/32078594/105649499-cf4b3680-5eb0-11eb-873e-9a70a98b21cf.png)

In conclusion, default is no default :-)

**Comment by rgriebl** [comment]
Please retest with the next dev build 229.

**Comment by paramecie** [comment]
ok

**Comment by paramecie** [comment]
Mmmm I can see no change !?

**Comment by paramecie** [comment]
Still bizarre!

This is a new document:
![image](https://user-images.githubusercontent.com/32078594/105655746-f6116900-5ec0-11eb-9ffb-a77f55a356fc.png)

Now I choose "BrickStore default" layout:
![image](https://user-images.githubusercontent.com/32078594/105655782-088ba280-5ec1-11eb-979e-9cce2af46869.png)

So, the default is still not the default :-)

**Comment by rgriebl** [comment]
New documents get the "User Default". That's probably different from the "BrickStore default" in your case. If you want to reset the user one, then load the BrickStore default, and save this layout with an empty name.

**Comment by paramecie** [comment]
Wow - did it, worked.

Don't expect users to do this or guess this (or read any f^cking doc) ;-)

Or add a "Reset my layout to BrickStore default" command somexhere and remove the "please leave it empty to restore..."?
Just a guess...

**Comment by paramecie** [comment]
Apart this, it's going to be a nice program, you can be proud.

Always those defects (but before see #113) and the abnormal columns title alignments (#29)

![image](https://user-images.githubusercontent.com/32078594/105657690-44c10200-5ec5-11eb-8bea-55d460607186.png)
